Cargo.toml

```
@@ -10,10 +10,18 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [dependencies]
-k8s-openapi = { version = "0.14.0", features = ["v1_20"] }
-kubewarden-policy-sdk = "0.3.2"
+k8s-openapi = { version = "0.11.0", features = ["v1_20"] }
```

I just learnt the `api` feature causes a longer compilation time and a bigger binary, but we don't actually need that inside of the policies. This introduces all the functions required to get/patch/delete Kubernetes objects

```suggestion
k8s-openapi = { version = "0.11.0", default_features = false, features = ["v1_20"] }
```